boto/ec2/elb/__init__.py

```diff
@@ -756,3 +757,71 @@ def detach_lb_from_subnets(self, name, subnets):
                                'Subnets.member.%d')
         return self.get_list('DetachLoadBalancerFromSubnets',
                              params, None)
+
+    # Tag methods
+
+    @staticmethod
+    def _build_tag_parameters(tags):
+        params = {}
+        for idx, key in enumerate(sorted(tags.keys()), start=1):
+            params['Tags.member.%d.Key' % idx] = key
+            value = tags[key]
+            if value is not None:
+                params['Tags.member.%d.Value' % idx] = value
+        return params
+
+    def get_all_tags(self, load_balancer_names=None):
```

Comment:
Attempting to call `get_all_tags()` without any load balancer names results in a 400 from AWS:
```
*** BotoServerError: BotoServerError: 400 Bad Request
<ErrorResponse xmlns="http://elasticloadbalancing.amazonaws.com/doc/2012-06-01/">
  <Error>
    <Type>Sender</Type>
    <Code>ValidationError</Code>
    <Message>1 validation error detected: Value null at 'loadBalancerNames' failed to satisfy constraint: Member must not be null</Message>
  </Error>
  <RequestId>df0a9c1a-fd9b-11e4-95e7-27e321da6c35</RequestId>
</ErrorResponse>
```

As of now, the ELB [describe-tags](http://docs.aws.amazon.com/cli/latest/reference/elb/describe-tags.html) endpoint only supports getting individual ELB's tags at a time.

`load_balancer_names` should probably be a required argument instead of a keyword argument.